<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMain</class>
 <widget class="QMainWindow" name="WndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FumoEdit-QT</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="TwMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="TabPostEditor">
       <attribute name="title">
        <string>Post Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QDateEdit" name="DePostDate"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="LblPostDate">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="LblPostTitle">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="LePostThumbName"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="LblPostInternalName">
          <property name="text">
           <string>Internal name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="LePostInternalName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="LblPostThumbName">
          <property name="text">
           <string>Thumbnail name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="LblPostID">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="LePostID"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="LblPostBody">
          <property name="text">
           <string>Body:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="6">
         <widget class="QPlainTextEdit" name="PtePostBody">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LePostTitle"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPictureManager">
       <attribute name="title">
        <string>Picture Manager</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>1. Add post pictures</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidget">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>70</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Variants</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Thumbnail</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>3. Add picture variants</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,1">
           <item row="0" column="1" rowspan="2" colspan="2">
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Image filename:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Label:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QTableWidget" name="tableWidget_2">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>71</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Label</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Image</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>2. Configure picture thumbnail</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,0,0,0,0,1">
           <item row="0" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Filename:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Offsets:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5" rowspan="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QGraphicsView" name="graphicsView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>388</width>
               <height>128</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>388</width>
               <height>128</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>.jpg</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="spinBox">
             <property name="suffix">
              <string>px</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Center</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Center</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="spinBox_2">
             <property name="suffix">
              <string>px</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu_I_think">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_post"/>
    <addaction name="action_Open_post"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menumenu_I_think"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New_post">
   <property name="text">
    <string>&amp;New post...</string>
   </property>
  </action>
  <action name="action_Open_post">
   <property name="text">
    <string>&amp;Open post...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>TwMain</tabstop>
  <tabstop>DePostDate</tabstop>
  <tabstop>LePostID</tabstop>
  <tabstop>LePostInternalName</tabstop>
  <tabstop>LePostTitle</tabstop>
  <tabstop>LePostThumbName</tabstop>
  <tabstop>PtePostBody</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
